hooks/post_gen_project.py

```diff
@@ -0,0 +1,17 @@
+from shutil import copyfile
+from os import listdir
+
+base_path_libs = '../libs'
+
+alldir = listdir(base_path_libs)
+
+libs = list(filter(lambda x: x.endswith('.jar'), alldir))
+
+for lib in libs:
+    path_lib = base_path_libs +'/'+ lib
+    destiny_lib = './libs/' + lib
+    copyfile(path_lib, destiny_lib)
+    print(' * Lib {} imported to android/libs/'.format(lib))
```

Comment:
Nitpicking: I believe it's best to use the wording `copied` here instead of `imported` -- because `import` is already an overloaded term here.